README.md

```diff
@@ -86,6 +86,7 @@ The above commands will start an interactive prompt. You can use the `arrow keys
 | --accessToken   | Github access token                       | string                   |
 | --all           | Show all commits                          | boolean (default: false) |
 | --branch        | Branch to backport to                     | string                   |
+| --githubUrl     | Url for Github API                        | string                   |
```

Comment:
nit: Terminology-wise I think "hostname" is more apt than "url".

Thinking out loud: is it conceivable that enterprise users might have different hostnames for git remote and the API? In that case it might make sense to have two separate options:

`--apiHostname` defaulting to `api.github.com`
`--gitHostname` defaulting to `github.com`

WDYT?

Reply:
I could see a case where that url formatting could change, it probably would be best to allow that kind of flexibility. Added the addition `apiHostname` option in 
0a3e845